packages/admin/cms-admin/src/generator/future/generator.ts

```
@@ -119,6 +120,11 @@ export async function runFutureGenerate(filePattern = "src/**/*.cometGen.ts") {
         const configs = await import(`${process.cwd()}/${file.replace(/\.ts$/, "")}`);
         //const configs = await import(`${process.cwd()}/${file}`);
 
+        const codeOuputFilename = `${targetDirectory}/${basename(file.replace(/\.cometGen\.ts$/, ""))}.tsx`;
+        await fs.rm(codeOuputFilename).catch((e) => false);
```

Should we really suppress errors like this?